packages/typespec-test/test/eventgrid_modular/generated/typespec-ts/src/api/models.ts

```diff
@@ -51,7 +51,7 @@ export interface LockToken {
   lockToken: string;
 }
 
-/** Details of the LockTokens information. This is used for both Acknowledge and Release operation response. */
+/** */
```

Comment:
Introduced by known issue, https://github.com/Azure/autorest.typescript/issues/1841.

Need to fix them in future.